packages/mobile/src/identity/verification.ts

```diff
@@ -100,13 +107,15 @@ export function* fetchVerificationState() {
     })
     if (timeout) {
       Logger.debug(TAG, '@fetchVerificationState', 'Token balances is null or undefined')
+      yield put(setVerificationStatus(VerificationStatus.Stopped))
```

Comment:
I would recommend putting this into a `finally` block to avoid needing it at every exit point in this function, and to handle `saga` cancellation.